src/faldbt/cp/events/types.py

```diff
@@ -0,0 +1,2869 @@
+# NOTE: COPIED FROM https://github.com/dbt-labs/dbt-core/blob/43edc887f97e359b02b6317a9f91898d3d66652b/core/dbt/events/types.py
```

Comment:
is this unreleased code?

Reply:
Only released for 1.0.0, so including it for backwards compatibilty

Reply:
I am now checking the dbt version to just avoid running the code that needs this in case we are pre-1.0.0.

This was necessary for 1.0.0 to run, but did not exist for pre-1.0.0 code. So, I included it and ran it every time. Now I only run it when we are using dbt 1.0.0 under the hood.